### ISSUE TITLE
Error in Express Form Submission Control Panel

### ISSUE THREAD

**Issue by superpikar** [solspace/craft-express-forms#10]
### Description
Hi Solspace team, 

I found an error when open express form >> submission in my staging server (Debian 9 - Nginx). But everything was OK on my development server (Windows 10 - XAMPP)
```
Unable to find component class 'Solspace\ExpressForms\Elements\Submission'.
```
![Capture](https://user-images.githubusercontent.com/2832280/56706115-f99d1c80-673d-11e9-867c-8e6d5c429de5.PNG)

even the Submission file is exist
![Capture](https://user-images.githubusercontent.com/2832280/56706441-33225780-673f-11e9-9526-7b72ac1924d0.PNG)

The suspect of the error is on `/templates/submissions/index.twig` line 6
```
{% set elementType = 'Solspace\\ExpressForms\\Elements\\Submission' %}
```

### Steps to reproduce

1. install express form on Debian 9
2. open express form >> submission

### Additional info

- Express Forms edition & version: 1.0.0-beta.3
- Craft version: 3.1.23
- PHP version: 7.2.0
- Database driver & version:
- Other plugins & versions (if applicable):

**Comment by kjmartens** [comment]
Thanks @superpikar. Is this happening in the Submissions list page or when viewing an individual submission? Would you be able to include the full stack trace in case that's helpful for our developer to locate the issue? :)

**Comment by superpikar** [comment]
It is happening when viewing Submissions list, I don't have any submission yet.

Sure, you can see the full stack trace below :)
![Capture](https://user-images.githubusercontent.com/2832280/56706872-c7d98500-6740-11e9-8421-8fa4bcb1561c.PNG)

**Comment by kjmartens** [comment]
This is now fixed in v1.0.0-beta.4 :)
